CHANGELOG

```diff
@@ -1,6 +1,51 @@
 devel
 -----
 
+* Support indexHint AQL option for traversals.
+  The structure of traversal index hints is as follows:
+  
+      { collection: { direction: { level: indexes } } }
+ 
+  with the following meanings:
+  - collection: the name of an edge collection for which the index hint should
+    be applied. Proper capitalization of the collection name is important.
+  - direction:  the direction for which to apply the index hint. Valid values
+    are `inbound` and `outbound`, in lowercase.
+  - level: the level/depth for which the index should be applied. Valid level 
+    values are the string `base` and any stringified numeric values greater
+    or equal to zero.
+
+  Example:
+
+      FOR v, e, p IN 1..5 OUTBOUND startVertex edgeCollection
+      OPTIONS { 
+        indexHint: {
+          "edgeCollection": {
+            "outbound": {
+              "base": ["edge"],
+              "1": "myIndex1",
+              "2": ["myIndex2", "myIndex1"],
```

Comment:
what does it mean if one specifies an array?

Reply:
It means the same as when specifying an array index hint for a collection `FOR` loop.
For example, one can already use something like
```
FOR doc IN collection OPTIONS { indexHint: ["index1", "index2"] }
...
```
even without this change. The candidate indexes for an index hint can be specified as a single string (one index, identified by its string name) or as an array of strings (multiple indexes, identified by their string names).
This is the same for traversal index hints: the candidate indexes for each level can be specified as a single string or as an array of strings.